static/inline/js/libs/player.js

```diff
@@ -0,0 +1,304 @@
+;(function(window,document) {
```

Comment:
Please take a look at the latest implementation of player settings on master (Solo or Carousel, etc)
With the integration of player 3.0 we are now relying in the player library to apply all these settings